### ISSUE TITLE
WebView2 re-installation fails if WebView2 is running

### ISSUE THREAD

**Issue by asjimene** [MicrosoftEdge/WebView2Feedback#3137]
**Description**

WebView2 fails to reinstall and leaves a partial broken install if a silent install is attempted of the same version of WebView2 and an application that utilizes WebView2 is running.

Log entries that indicate the error:
```
[6040:9932:0117/212641.135:981885687:VERBOSE1:set_reg_value_work_item.cc(199)] Successfully wrote value Can not install the same Microsoft Edge version that is currently running. Please close Microsoft Edge and try again. into Software\Microsoft\EdgeUpdate\ClientState\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}
[6040:9932:0117/212641.135:981885687:VERBOSE1:work_item_list.cc(53)] Successful execution of work item list Write Installer Result
[6040:9932:0117/212641.135:981885687:VERBOSE1:setup_main.cc(2762)] Deleting temporary directory C:\Program Files (x86)\Microsoft\EdgeWebView\Temp
[6040:9932:0117/212641.135:981885687:VERBOSE1:google_update_settings.cc(690)] Installer failed. Archive type: 3. Return code: 31
```

**Version**
Runtime: Stable 109.0.1518.55
OS: Windows 11

**Repro Steps**

Install WebView2 as administrator with the standalone (or bootstrap installer) with ```/install /silent```
Open an application that utilizes WebView2 like Citrix Workspace
Install WebView2 as administrator with the silent install command ```/install /silent```

Install will fail and leave a partial broken installation of WebView2. Expected behavior would be to return that the product is already installed.

**Screenshots**
![image](https://user-images.githubusercontent.com/3790176/213244687-9869c87b-7d60-4665-a94f-a88e01e9f5a1.png)

**Additional context**
This issue hit us when we messed up detection in Intune and ConfigMgr for a WebView2 deployment. The WebView2 application was not being detected after a successful installation, so installation was attempted again, which left many endpoints in a broken state.

**Comment by sln162** [comment]
![image](https://user-images.githubusercontent.com/11196986/213248822-53c5218d-c909-4f39-bc84-808d29facdfa.png)
![image](https://user-images.githubusercontent.com/11196986/213249047-eb657911-8b19-4c49-8f37-8c8203fbdb69.png)
According to what you said, the manual execution is the same prompt, and there is indeed a file loss.

**Comment by sln162** [comment]
Version 109.0.1518.61 of WebView2 seems to fix this issue

**Comment by LiangTheDev** [comment]
The issue should have been fixed in 109.0.1518.61.

**Comment by asjimene** [comment]
Yes, we are seeing this fixed in the latest build.
